src/modules/flow/iio/nodes.c

```diff
@@ -904,6 +909,11 @@ light_open(struct sol_flow_node *node, void *data, const struct sol_flow_node_op
 
     mdata->iio_base.config.buffer_size = opts->buffer_size;
     mdata->iio_base.config.sampling_frequency = opts->sampling_frequency;
+    ret = snprintf(mdata->iio_base.config.sampling_frequency_name,
+        sizeof(mdata->iio_base.config.sampling_frequency_name), "%s", "in_light_");
```

Comment:
This should be `in_illuminance_` and not `in_light_`.
You could refer all iio channel type name here:
http://lxr.free-electrons.com/source/drivers/iio/industrialio-core.c#L54